examples/iso/k3s-single-node.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: 1.0
+image:
+  arch: x86_64
+  imageType: iso
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso
+  outputImageName: single-node-k3s-no-selinux.iso
+kubernetes:
+  version: v1.28.7+k3s1
+  network:
+    apiVIP: 192.168.100.19
+  nodes:
+    - hostname: node1.suse.com
+      type: server
```

Comment:
I'd drop this. It's ignored for single nodes.

Reply:
This meaning the type attribute right, not the entire node reference?

Reply:
No - the entire `nodes` section.